README.md

```diff
@@ -218,6 +218,10 @@ Thus, all calculations happen inside the XSLT files. We decided to implement
 it this way after a less successful attempt to do it all in Java. It seems
 that XSL is much more suitable for manipulations with data than Java.
 
+## Known Limitations
+
+* The java compiler is known to inline constant variables as per [JLS 13.1](https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.1). This may affect the results obtained by some metrics when a method of a class being analyzed accesses a constant variable (basically the access to the constant will not be included in the calculations).
```

Comment:
@llorllale I think we can drop the "of a class being analyzied" -- it is understood that the method belongs to a class which is being analyzed. And I would drop the "basically" -- it's the kind of thing one says when they are not sure about something and we don't want to give that feeling :) 

what do you think?

Reply:
@amihaiemil sure, I'll make the changes and let you know